asv_bench/benchmarks/io/csv.py

```diff
@@ -318,7 +318,8 @@ def setup(self, sep, decimal, float_precision):
             "".join([random.choice(string.digits) for _ in range(28)])
             for _ in range(15)
         ]
-        rows = sep.join([f"0{decimal}" + "{}"] * 3) + "\n"
+        row = "".join([f"0{decimal}", "{}"])
```

Comment:
Thanks for working on this! 

Would it work to do
```python
rows = sep.join([f"0{decimal}{{}}"] * 3) + "\n"
```
?

Reply:
Thank you, it works nicely. I corrected my PR.